clar2wasm/src/words/conditionals.rs

```diff
@@ -256,4 +347,43 @@ mod tests {
             eval("(list 3 4)"),
         );
     }
+
+    #[test]
+    fn and() {
+        assert_eq!(
+            eval(
+                r#"
+(define-data-var cursor int 6)
+(and
+  (var-set cursor (+ (var-get cursor) 1))
+  true
+  (var-set cursor (+ (var-get cursor) 1))
+  false
+  (var-set cursor (+ (var-get cursor) 1)))
+(var-get cursor)
+                "#
+            ),
+            eval("8")
```

Comment:
(This is not blocking the merging of this PR, but just a comment.)
I like the simplicity of this technique, but it messes up my debugging technique of uncommenting this line in tools.rs and checking the test.wasm. 
```rs
compile_result.module.emit_wasm_file("test.wasm").unwrap();
```

Maybe we should add some better mechanism for debugging in the testing helper functions.